ClosedXML/Utils/OpenXmlHelper.cs

```diff
@@ -11,7 +11,7 @@ public static BooleanValue GetBooleanValue(bool value, bool defaultValue)
 
         public static bool GetBooleanValueAsBool(BooleanValue value, bool defaultValue)
         {
-            return value == null ? defaultValue : value.Value;
+            return value.HasValue ? value.Value : defaultValue;
```

Comment:
Again, ```BooleanValue``` is a class, not a Nullable<T>. So the ```value``` variable can be ```null```, and with this ```value.HasValue``` will trigger ```NullReferenceException```. You should check both if ```value != null``` AND ```value.HasValue```.